feat(sidebar):Implemented sidebar navigation for Practitioner dashboa…

OVER-CODER opened the pull request:
Description:

Fix #15 

This PR implements the sidebar navigation structure for practitioners after successful login and terms acceptance. The sidebar serves as the primary navigation element within the practitioner interface and includes the following tabs:
- Dashboard (default view)
- Waiting Room
- Open Consultations
- Closed Consultations
- Invites
- Profile

Key Features:

- Active tab highlighting to improve navigation experience.
- Responsive and accessible design.

![Screenshot from 2025-04-09 23-29-06](https://github.com/user-attachments/assets/22c275b5-40e0-413d-ba0d-04cf02b2a5fa)


This enhances usability by providing clear and structured access to core functionalities for practitioners.

gorg16 commented on practitioner/src/app/components/sidebar/sidebar.component.html:
```diff
@@ -0,0 +1,21 @@
+            <i class="fas {{ item.icon }}"></i>
+          </div>
+          <div class="label">{{ item.label }}</div>
+          <div class="badge" *ngIf="item.badge">{{ item.badge }}</div>
```
Create component for badge

gorg16 commented on practitioner/src/app/layouts/main-layout/main-layout.component.scss:
```diff
@@ -0,0 +1,123 @@
+  }
+}
+
+.header-center {
```
rename this classes

gorg16 commented on practitioner/src/app/layouts/main-layout/main-layout.component.scss:
```diff
@@ -0,0 +1,123 @@
+  }
+}
+
+.header-right {
```
rename this classes

gorg16 commented on practitioner/src/index.html:
```diff
@@ -6,6 +6,8 @@
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
```
Do not use CDN links. Use local files

OVER-CODER commented:
I have made the required changes, kindly review them.

gorg16 commented:
@OVER-CODER please fix conflicts

OVER-CODER commented:
@gorg16 I have resolved all the conflicts. Sidebar and Dashboard works perfectly now.

![Screenshot from 2025-04-11 16-54-12](https://github.com/user-attachments/assets/2902dad1-1b25-4de5-878c-b8e6c99d6844)

OVER-CODER commented:
Hey! @gorg16, I have fixed the ui as per requirements. Kindly review and lemme know if any other changes are required.

- Fixed the badge location
- Fixed the active state of sidebar.
- used proper icon packs.

![Screenshot from 2025-04-12 14-01-44](https://github.com/user-attachments/assets/ad1d8c09-a8be-4442-9c18-43de7e7957b7)

Thankyou!
